tests/main/interfaces-bluetooth-control/task.yaml

```diff
@@ -0,0 +1,57 @@
+summary: check that the bluetooth-control interface works
+
+systems: [ubuntu-core-16-arm-64]
+
+prepare: |
+    echo "Given a snap declaring a plug on bluetooth-control is installed"
+    . $TESTSLIB/snaps.sh
+    install_generic_consumer bluetooth-control
+
+restore: |
+    rm -f *.error version class control
+
+execute: |
+    CONNECTED_PATTERN=":bluetooth-control +generic-consumer"
+    DISCONNECTED_PATTERN="^\- +generic-consumer:bluetooth-control"
+    BTDEV="$(find /sys/devices/ -type d -name bluetooth)"
+
+    echo "Then the plug is disconnected by default"
+    snap interfaces | MATCH "$DISCONNECTED_PATTERN"
+
```

Comment:
Done, in fact the test is currently only enabled on `ubuntu-core-16-arm-64` (dragonboard is the only reference platform that has bt hardware) but having this in place is very good in case we enable it elsewhere.